Automattic/wp-calypso
# Seller Experience: add blocks to frontpage

#### Changes proposed in this Pull Request

Send the user to a new Front Page with payment pattern after the seller flow.

#### Testing instructions



* Access `start/setup-site/intent?siteSlug=<SITE SLUG>&flags=seller-experience` (which enables the `seller-experience` feature flag)
* Click the "Start Selling" button
* Click "Continue"
* Start Simple -> Continue
* The editor should open with a new page containing the payment block pattern. 



Related to #60731
